CONTRIBUTING.md

```diff
@@ -154,6 +140,10 @@ To contribute operations to the mobility database catalogs, it is suggested that
 
 This project uses [the Black code formatter](https://github.com/psf/black), which is compliant with [PEP8](https://www.python.org/dev/peps/pep-0008/), the style guide for Python code. A [pre-commit](https://pre-commit.com/) hook file is provided with the repo so it is easy to apply Black before each commit. A CI workflow is testing the code pushed in a pull request to make sure it is PEP8 compliant.
 
+## Integration Tests
+
+In order to avoid invalid sources in the Mobility Database Catalogs, any modification made in the repository, addition or update, must pass the integration tests before being merged into the project. The integration tests are listed in the [Test Integration](/tests/test_integration.py) module.
```

Comment:
Would it be useful to describe each test? Perhaps in another documentation file?

Reply:
> @maximearmstrong It would be helpful to get your perspective on how to write CONTRIBUTING.md so it's easy for a contributor to find the tests and run them on their code changes.

Also, is it here that we should describe how to run the tests? Or did you mean all tests, not only integration?

Reply:
@maximearmstrong It's your call if you think another documentation file is needed. 

I wasn't sure of the best approach here - we could just reference the tests in README and have a separate documentation file that we link to if that's the best course of action.

Reply:
> I wasn't sure of the best approach here - we could just reference the tests in README and have a separate documentation file that we link to if that's the best course of action.

I agree with this, I think it would be the best.

Reply:
I will create an issue for this - for now I'll move the integration tests section back to the README.